6-Typescript/server/interfaces/DB.ts

```diff
@@ -0,0 +1,10 @@
+import { Item } from './Item'
+import { Guid } from './Guid'
+export interface DB {
+    connect: () => Promise<boolean>
+    disconnect: () => Promise<boolean>
+    getAllItemsFromDB: (userID: Guid) => Promise<Item[]>
+    getItemFromDB: (id: Guid) => Promise<Item>
+    setItemInDB: (userID: Guid, _id: Guid, title: string, content: string) => Promise<Item>
```

Comment:
_id is the ID of the items stored by mongo (I have to override it so mongo wont store it as an ObjectID object). i generelly use _id if i want to use { _id, userID, title, content} inside the function, instead of 
{
    _id: id,
   userID: userID,
   title: title,
   content: content
}
do you still want me to changes this?